README.md

```diff
@@ -12,6 +12,17 @@ Monorepo to hold all shareable linting and formatting configurations.
 
 When using both ESLint and Prettier in a project, it's nice to have files automatically have ESLint errors fixed on save as well as have non JavaScript/TypeScript files formatted on save. This section explains how to set that up in an editor.
 
+### Versions Bump
+Patch releases: 
+  - Bump dependencies to patch or minor releases
+  - Fixs not impacting the configs or adding new rules
+Minor releases: 
+  - Adding exceptions to rules (ex: disable a default rule)
+  - Changin Minor rules (Nothing that will reformat code by default)
+Major releases: 
+  - Bump dependencies to major releases
+  - Changing rules that will reformat code by default (ex: `{ printWidth: 80 }` -> `{ printWidth: 120 }`)
```

Comment:
The preview doesn't looks good, you need new lines

https://github.com/Mixgenius/linting-and-formatting/pull/28/files#diff-04c6e90faac2675aa89e2176d2eec7d8